src/Hl7.Fhir.Support.Poco/Model/Generated/Code.cs

```diff
@@ -71,6 +71,7 @@ public Code(string value)
     [FhirElement("value", IsPrimitiveValue=true, XmlSerialization=XmlRepresentation.XmlAttr, InSummary=true, Order=30)]
     [DeclaredType(Type = typeof(SystemPrimitive.String))]
     [DataMember]
+    [CodePattern]
```

Comment:
This is an addition to generated code. I don't see a PR for `FirelyTeam/fhir-codegen`. Is this correct?

Reply:
Right! Here it is: https://github.com/FirelyTeam/fhir-codegen